code/modules/power/supermatter/supermatter.dm

```diff
@@ -382,7 +390,11 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	else
 		// Pass all the gas related code an empty gas container
 		removed = new()
-
+	LAZYCLEARLIST(active_overlays)
+	if(psyCoeff > 0)
+		psyOverlay.alpha = psyCoeff * 255
+		LAZYADD(active_overlays, psyOverlay)
+	overlays = active_overlays
```

Comment:
Instead of keeping an `active_overlays` list here use a boolean to keep track on if `psyOverlay` is in `overlays` or not, and add or remove it as appropriate.
That way we don't override other existing overlays if they come to be in the future, and don't have to maintain two different lists.

Reply:
Sure